content/en/getting_started/synthetics/api_test.md

```diff
@@ -0,0 +1,62 @@
+## Create an API test
+
+[API tests][1] help you monitor your API endpoints and alert you when they are failing or too slow. These checks verify that your applications are responding to requests and meet any conditions you define, such as response time, HTTP status code, and header or body contents. Use the [Datadog API][2] to see the full list.
```

Comment:
first link seems broken as well

Reply:
Whoops, yes. And the headers are all off as my changes didn't save. Will add those in.